packages/Breadcrumb/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { cloneElement, forwardRef } from 'react'
+import { RightIcon } from '@welcome-ui/icons.right'
+import { node, oneOfType, string } from 'prop-types'
+
+import { COMPONENT_TYPE } from '../../src/utils/propTypes'
+
+import { Item } from './Item'
+import * as S from './styles'
+
+export const Breadcrumb = forwardRef(
+  ({ as = 'a', children, separator = <RightIcon size="sm" />, ...rest }, ref) => {
```

Comment:
Not a fan of `as` here since it's actually used on the children and not the element itself, what about `renderChildrenAs`, I know that it's a mouthful but at least it's clear 😛

Reply:
👍